src/acceptance/acceptance_suite_test.go

```diff
@@ -0,0 +1,88 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestAcceptance(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Acceptance Suite")
+}
+
+var (
+	dora        = "dora"
+	doraWindows = "dora-windows"
```

Comment:
Why not make the app names constants if you don't plan on editing them? Can you make the constant var names more clearly app names as well.